tapchannel/aux_leaf_signer.go

```diff
@@ -241,43 +244,49 @@ func (s *AuxLeafSigner) processAuxSigBatch(chanState *channeldb.OpenChannel,
 	defer s.Wg.Done()
 
 	log.Tracef("Processing %d aux sig jobs", len(sigJobs))
-
 	for idx := range sigJobs {
 		sigJob := sigJobs[idx]
-		cancelAndErr := func(err error) {
+		respondErr := func(err error) {
 			log.Errorf("Error processing aux sig job: %v", err)
 
-			close(sigJob.Cancel)
 			sigJob.Resp <- lnwallet.AuxSigJobResp{
```

Comment:
Wherever we send to a channel, we should also read on the `Quit` channel to make sure we never block on a send (even if it's buffered).

Reply:
hmm, so if we receive `Quit` we abandon sending the err? that may then block `lnd` then IIUC.

Also, if we select on `Quit`, and we entered this function from a `Quit` case statement, we would never send the error, since we're reading an already closed channel.

Reply:
Hmm, yeah, you're right. We have to assume the `Resp` channel is buffered here though (which it is), then this should never block.

Reply:
> We have to assume the Resp channel is buffered here though (which it is), then this should never block.

Yeah this is what was puzzling w.r.t the original stack trace that led us down this line of inquiry: unless some weird mutation happened, why would `tapd` block on the channel send?